cumulus/parachains/runtimes/testing/penpal/src/xcm_config.rs

```diff
@@ -316,9 +361,47 @@ pub type Reserves = (
 	NativeAssetFrom<SystemAssetHubLocation>,
 	AssetPrefixFrom<EthereumLocation, SystemAssetHubLocation>,
 );
+
 pub type TrustedTeleporters =
 	(AssetFromChain<LocalTeleportableToAssetHub, SystemAssetHubLocation>,);
 
+parameter_types! {
+	pub SiblingBridgeHubWithEthereumInboundQueueInstance: Location = Location::new(
+				1,
+				[
+					Parachain(bp_bridge_hub_rococo::BRIDGE_HUB_ROCOCO_PARACHAIN_ID),
```

Comment:
Same here, these values need to configurable so that we can set them from any runtime. Here's an example of how the Ethereum network ID was made configurable in Penpal: https://github.com/paritytech/polkadot-sdk/pull/3564

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/114#discussion_r1560351339